plugins/pagerduty/komand_pagerduty/connection/connection.py

```diff
@@ -20,25 +17,11 @@ def connect(self, params={}):
         """
 
         key = params.get("api_key").get("secretKey")
```

Comment:
Same for the `Input` imports here too 😃

Reply:
this has been updated to use the input object